openedx/frontend-app-course-authoring
# fix: files page timeout

JIRA Ticket: [TNL-11255](https://2u-internal.atlassian.net/browse/TNL-11255)

> The Files page for large courses takes over 30 seconds to load.  Notice the page size is 13165, and the call eventually fails…

This PR adds missing failure messages to users when the initial fetch of the files and videos fails. Additionally this PR breaks up the initial fetch of the files page into 50 file increments (akin to fetching one page at a time). This segmenting will allow the first page to load quickly and prevent API timeout errors.

Testing
1. Navigate to the Import page
2. Import a course that has over 1000 assets
3. Once the import has completed, navigate to the Files page
4. Page should fetch the first 50 assets (look at the table header for this information)
5. Should continue to fetch assets until all are loaded
   - This can be observed via the header "Showing 50 of {total fetched} or the addition of pages to pagination footer
